src/features/donation/components/cross-chain-amount-entry.tsx

```
@@ -488,10 +488,24 @@ export const CrossChainAmountEntry: React.FC<CrossChainAmountEntryProps> = ({
         </div>
       </div>
-      {isDisabled && (
-        <div className="text-sm text-red-500">
-          Please enter a valid amount in {selectedTokenData?.symbol || "USDC"} greater than an
-          equivalent of 0.1 NEAR.
+      {isDisabled && price > 0 && nearPrice > 0 && (
+        <div className="flex items-center gap-2 text-sm text-red-500">
+          <span>
+            Please enter a valid amount in {selectedTokenData?.symbol || "USDC"} greater than an
+            equivalent of 0.1 NEAR
+            {" "}(min: {((0.1 * nearPrice) / price).toFixed(4)} {selectedTokenData?.symbol || "USDC"}).
```

ESLint

[error] Replace `⏎············{"·"}(min:·{((0.1·*·nearPrice)·/·price).toFixed(4)}` with `·(min:·{((0.1·*·nearPrice)·/·price).toFixed(4)}{"·"}⏎···········`

(prettier/prettier)